translations/pt_BR/README.md

```diff
@@ -0,0 +1,336 @@
+# IPFS é a Web Distribuída
+
+> Um protocolo hipermídia ponto a ponto para tornar a web mais rápida, segura e aberta.
+
+[![ipfs](https://img.shields.io/badge/irc-%23ipfs-brightgreen.svg)](https://webchat.freenode.net/?channels=ipfs)
+[![Changelog #204](https://img.shields.io/badge/changelog-%23204-lightgrey.svg)](https://changelog.com/204)
+
+Bem vindo ao IPFS! Por que não assiste um [vídeo de demonstração](https://www.youtube.com/watch?v=8CMxDNuuAiQ) para começar? Por favor, poste perguntas e idéias em https://discuss.ipfs.io
+
+## Índice
+
+- [Visão Geral](#overview)
+  - [Resumo](#quick-summary)
+- [Como IPFS funciona](#how-ipfs-works)
+  - [IPFS Papers](#ipfs-papers)
+  - [IPFS Conversas](#ipfs-talks)
+- [Mais Sobre IPFS](#more-about-ipfs)
+  - [Estado atual do IPFS](#current-state-of-ipfs)
+  - [Distribuição Alpha](#alpha-distribution)
+  - [Questões de Segurança e Divulgações](#security-issues-and-disclosures)
+- [Projeto e Comunidade](#project-and-community)
+- [Links do Projeto](#project-links)
+  - [Implementações do Protocolo](#protocol-implementations)
+  - [Bibliotecas de Clientes HTTP](#http-client-libraries)
+  - [Diretório de Projeto](#project-directory)
+  - [Outros recursos da comunidade](#other-community-resources)
+- [Licença](#license)
+
+## Visão Geral
+
+IPFS ([the InterPlanetary File System](https://github.com/ipfs/faq/issues/76)) é um novo protocolo de distribuição hipermídia, endereçado por conteúdo e identidades. O IPFS permite a criação de aplicativos completamente distribuídos. O objetivo é tornar a web mais rápida, segura e aberta.
+
+O IPFS é um sistema de arquivos distribuídos que procura conectar todos os dispositivos computacionais com o mesmo sistema de arquivos. De certa forma, isso é semelhante aos objetivos originais da Web, mas o IPFS é mais similar a um único bittorrent swarm trocando objetos git. Você pode ler mais sobre suas origens no documento [IPFS - Content Addressed, Versioned, P2P File System](https://github.com/ipfs/ipfs/blob/master/papers/ipfs-cap2pfs/ipfs-p2p-file-system.pdf?raw=true).
+
+O IPFS está se tornando um novo grande subsistema da Internet. Se construído corretamente, pode complementar ou substituir o HTTP. Pode complementar ou substituir. Parece loucura. _É_ uma loucura.
+
+[![Alpha Demo](../../img/ipfs-alpha-video.png)](https://www.youtube.com/watch?v=8CMxDNuuAiQ)
+
+Quer ver mais? Confira a palestra de Juan Benet no Sourcegraph - [IPFS: The Permanent Web](https://www.youtube.com/watch?v=Fa4pckodM9g).
+
+### Sumário
+
+O IPFS é um protocolo que:
```

Comment:
As a non-speaker - is the leading "O" for stylistic or linguistic purposes? (aka does "IPFS é um protocolo que" mean "IPFS is a protocol that:" - or is it ungrammatical?

Reply:
"O" is for linguistic purposes. 

Everything is "he" or "she" in Portuguese. "O" is equal to "he", so, he (IPFS) is an protocol.

Reply:
Ok - just checking. I saw it for some bullets but not others, so was a little confused...